tests/features/banner.feature

```diff
@@ -0,0 +1,95 @@
+@api
+Feature: Banner paragraph.
+  As a content editor
+  I need to be able to use banner paragraphs
+  so I can convey important messages to the audience.
+
+  @cleanup:media
+  Scenario: Banner paragraph creation.
+    Given I am logged in as a user with the "create oe_demo_landing_page content, access content, edit any oe_demo_landing_page content, create image media" permission
+    # Create an "Image" media.
+    When I go to "the image creation page"
+    And I fill in "Name" with "My Image 1"
+    And I attach the file "media/example_1.jpeg" to "Image"
+    And I press "Upload"
+    And I fill in "Alternative text" with "Image Alt Text 1"
```

Comment:
Use here media creation steps we used with events, ask @nagyad for more details.